includes/admin/admin-commands.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * SCF Commands Integration
+ *
+ * @package Secure Custom Fields
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Initializes SCF commands integration
+ *
+ * This function handles the integration with WordPress Commands (Cmd+K / Ctrl+K),
+ * providing navigation commands for SCF admin pages and custom post types.
+ *
+ * The implementation follows these principles:
+ * 1. Only loads in screens where WordPress commands are available.
+ * 2. Performs capability checks to ensure users only see commands they can access.
+ * 3. Core administrative commands are only shown to users with SCF admin capabilities.
+ * 4. Custom post type commands are conditionally shown based on edit_posts capability
+ *    for each specific post type.
+ * 5. Post types must have UI enabled (show_ui setting) to appear in commands.
+ *
+ * @since 6.5.0
+ */
+function acf_commands_init() {
+	// Only load on admin screens.
+	if ( ! is_admin() ) {
+		return;
+	}
+
+	// Ensure we only load our commands where the WordPress commands API is available.
+	if ( ! wp_script_is( 'wp-commands', 'registered' ) ) {
```

Comment:
is this really necessary?

Reply:
This is to avoid trying to access the command palette functions in WP versions earlier than 6.3 or without Gutenberg, in which cases my PR should do nothing.

Reply:
ok I didn't realize we support old versions like that in scf